### ISSUE TITLE
Basic Content Types

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa01#70]
Module to hold basic content types that will be used on most sites.

**Comment by pyrello** [comment]
What are your thoughts on adding layout builder support into these content types? Do we want to allow for News, Events, and Seminars to have their display handled by layout builder? Do we want to control it at the content type level? Or allow for customization of the individual nodes? I bring this up because I was looking into adding this to pages in #92.

**Comment by bspeare** [comment]
I think it would be useful to have the ability to override and customize an individual News or Event page.  Every so often have departments that have News and Events that would benefit from the ability to make minimal adjustments to the layout.

**Comment by pyrello** [comment]
I agree with you, @bspeare. I’m looking into how this can be accomplished via the modules .install files rather than added to config. This is because, currently, it looks like the layout builder system expects a field called ‘layout_builder__layout’ which is where all the layout-related stuff is attached to an entity. I tried an experiment where I created a field with a different name in config, but could not get it to work.
